_events/sr2020/london-tech-day-november.md

```diff
@@ -0,0 +1,47 @@
+---
+title: London November Tech Day
+date: 2019-11-30 10:00:00
+layout: event
+type: techday
+location: Thread, Whitechapel
+---
+
+Tech Days are opportunities for teams to get help if they're struggling with a
+technical issue, and is a controlled environment for students to make more
+progress on their robots. It's also an opportunity to see how well they're doing
+in comparison to other teams.
+
+London Tech Days will only run of there is sufficient interest. Spaces are
+limited to six teams, so be sure to [let us know][teams-contact] if you'd like
+to attend. For this Tech Day, please express your interest by Wednesday 27th
+November.
+
+**Please bring your laptops!**
+
+## Directions
+
+The London Kickstart is being held in the [offices of Thread][venue-map] at:
+
+1 Alie Street,  
+Whitechapel,  
+London,  
+E1 8DE
+
+The easiest way to travel will be by public transport -- Aldgate, Aldgate East
+and Tower Gateway stations are all within a five minute walk, and Liverpool
+Street station is a ten minute walk away. There are also a large number of
+busses which stop nearby.
```

Comment:
```suggestion
buses which stop nearby.
```

https://www.grammarly.com/blog/busses-buses/

Reply:
"busses" is not wrong so I'm going to leave this.